cla_public/apps/contact/fields.py

```diff
@@ -76,7 +80,9 @@ def process_formdata(self, valuelist):
 
     @classmethod
     def _format(cls, value):
-        return '{:%Y%m%d}'.format(value)
+        if value and not isinstance(value, basestring):
```

Comment:
if you're going to do `isinstance` here, why not make it `isinstance(value, datetime.datetime)` (or `datetime.date`)